src/utilities/theme/color-adjustments.ts

```diff
@@ -175,7 +175,7 @@ export const colorAdjustments: ColorAdjustments = {
   iconSubduedOnSurface: {
     baseColor: 'onSurface',
     light: {lightness: 59.8},
-    dark: {lightness: 88},
+    dark: {lightness: 62},
```

Comment:
We'll need to review this, for now, I made it the same as the text in dark mode.